merino-cache/src/redis/mod.rs

```diff
@@ -79,6 +79,7 @@ impl SimpleRedisLock {
     /// See if a record update is locked for pending update.
     ///
     /// This does not check lock value, only if a lock exists.
+    #[allow(clippy::wrong_self_convention)]
```

Comment:
We could rename this to something like `can_lock()`, but it really doesn't sound better than `is_locked()`. So I've decided to silence this warning.

Reply:
What do you think? @pjenvey

Reply:
Definitely silence it, it's a bug they've already fixed: https://github.com/rust-lang/rust-clippy/pull/8738 :)